packages/ses/src/module-proxy.js

```diff
@@ -25,7 +25,14 @@ const q = JSON.stringify;
 // module namespace proxy to behave properly, so it throws exceptions until
 // after the compartment has executed the module.
 // The module instance must develop the proxied exports during execution,
-// seal them, and only then activate the module proxy.
+// freeze them, and only then activate the module proxy.
+//
+// The module exports proxy's behavior differs from the ECMAScript 262
+// specification for "module namespace exotic objects" only in that value
+// property descriptors have a non-writable "value".
```

Comment:
I haven't tried, but my read of your code is they don't have a "value" at all, but are visibly accessor properties with an available getter. We should try and see.

Reply:
Pardon. Fixed the referant!